src/psyclone/tests/psyir/frontend/fparser2_fortran_use_test.py

```diff
@@ -46,34 +46,67 @@
 from psyclone.psyir.frontend.fparser2 import Fparser2Reader
 from psyclone.psyir.nodes import KernelSchedule, Container
 from psyclone.psyir.symbols import ContainerSymbol, SymbolError, Symbol, \
-    DataSymbol, AutomaticInterface, INTEGER_SINGLE_TYPE
+    DataSymbol, AutomaticInterface, INTEGER_SINGLE_TYPE, ScalarType, \
+    RoutineSymbol
 
 
 def test_use_return(parser):
```

Comment:
Apologies that I missed this before but can you use the `fortran_reader` fixture here rather than having to setup a reader and everything?

    psyir = fortran_reader.psyir_from_source(code)

Reply:
Done